docs/exceptions.md

```diff
@@ -224,31 +224,38 @@ Conclusion: we might want to catch unexpected errors in Node, but there are plen
 
 ## Unexpected failures in browser
 
-In case of a browser restarting (reloading the page) usually considered as an awful behavior from
-the UX point of view, so it might be not an option. We may choose not to restart in a hope of
-providing a better UX at a risk of leaking inconsistent state to the database etc. Some bugs are
+Just as in Node we could "restart" the browser or reload the page. However, that option is usually considered as an awful behavior from
```

Comment:
I don't wan't to propose restarting the browser 😄  
I meat that in browser `restarting == reloading the page`